src/components/biggive-timeline-entry/biggive-timeline-entry.tsx

```diff
@@ -6,11 +6,14 @@ import { Component, Prop } from '@stencil/core';
   shadow: true,
 })
 export class BiggiveTimelineEntry {
-  @Prop() entryDate: string;
+  /**
+   * Should be in the format 'YYYY-mm-dd', for example '2023-01-01'
+   */
+  @Prop() date: string;
 
-  @Prop() entryYear: string;
+  @Prop() tab: string;
 
-  @Prop() entryTitle: string;
+  @Prop() heading: string;
```

Comment:
What's the difference between `tab` and `heading`? Might be useful to have an example of a suitable values for each one. Or is `tab` not used? I'm not seeing usages of `tab`.

Reply:
Sorry - tab can be removed, I have removed now